Content.Shared/Storage/Components/MagnetPickupComponent.cs

```diff
@@ -16,7 +16,7 @@ public sealed partial class MagnetPickupComponent : Component
     /// What container slot the magnet needs to be in to work.
     /// </summary>
     [ViewVariables(VVAccess.ReadWrite), DataField("slotFlags")]
```

Comment:
This is an old syntax. `ViewVariables(VVAccess.ReadWrite)` is now included as a part of `DataField`. `DataField` also now automatically creates an alias in camelcase. Since you're here, could you also please clean up this component a lil?

```suggestion
    [DataField]
```